src/server/components/dashboard/controllers/listExportController.ts

```diff
@@ -0,0 +1,91 @@
+import type { NextFunction, Request, Response } from "express";
+import type { ListItem } from "server/models/types";
+import type { ListItemJsonData } from "server/models/listItem/providers/deserialisers/types";
+
+import { prisma } from "server/models/db/prisma-client";
+import { json2csv } from "json-2-csv";
+import { Readable } from "node:stream";
+import { startCase } from "lodash";
+import { format } from "date-fns";
+import { logger } from "server/services/logger";
+import { AuditEvent } from "@prisma/client";
+
+type ListItemWithJsonData = ListItem & { jsonData: ListItemJsonData };
+
+export async function listExportController(req: Request, res: Response, next: NextFunction) {
+  try {
```

Comment:
Generally if an entire handler is wrapped in a try/catch block and there's no specific handling that's being done, then it's probably best handled on a higher level. Throwing or passing in an error to `next` will be handled in `server/middlewares/error-handlers.ts`.